msm/src/serialization/witness.rs

```diff
@@ -156,46 +160,119 @@ pub fn deserialize_field_element<Fp: Field, Env: InterpreterEnv<Fp>>(
     }
 
     // FIXME: range check
-    let c0 = Env::get_column_for_msm_limb(0);
-    let c1 = Env::get_column_for_msm_limb(1);
-    let c2 = Env::get_column_for_msm_limb(2);
-    let c3 = Env::get_column_for_msm_limb(3);
-    let c4 = Env::get_column_for_msm_limb(4);
-    let c5 = Env::get_column_for_msm_limb(5);
-
-    env.bitmask_be(&input_limb0, 15, 0, c0);
-    env.bitmask_be(&input_limb0, 30, 15, c1);
-    env.bitmask_be(&input_limb0, 45, 30, c2);
-    env.bitmask_be(&input_limb0, 60, 45, c3);
-    env.bitmask_be(&input_limb0, 75, 60, c4);
-    {
+    let c0_var = {
+        let c0 = Env::get_column_for_msm_limb(0);
+        env.bitmask_be(&input_limb0, 15, 0, c0)
+    };
+    let c1_var = {
+        let c1 = Env::get_column_for_msm_limb(1);
+        env.bitmask_be(&input_limb0, 30, 15, c1)
+    };
+    let c2_var = {
+        let c2 = Env::get_column_for_msm_limb(2);
+        env.bitmask_be(&input_limb0, 45, 30, c2)
+    };
+    let c3_var = {
+        let c3 = Env::get_column_for_msm_limb(3);
+        env.bitmask_be(&input_limb0, 60, 45, c3)
+    };
+    let c4_var = {
+        let c4 = Env::get_column_for_msm_limb(4);
+        env.bitmask_be(&input_limb0, 75, 60, c4)
+    };
+    let c5_var = {
```

Comment:
So the 5th is inlined because 88 is not perfectly divisible by chunks of 15bits?

Reply:
Yes.